.github/workflows/tests.yaml

```diff
@@ -21,6 +21,7 @@ jobs:
       run: |
         pip install cython numpy  # needed for pycocotools
         pip install .[datasets,tests,train]
+        pip install git+https://github.com/kornia/kornia@master # required for mypy fixes
```

Comment:
This might need to be first to prevent installing twice

Reply:
Done